packages/zosfiles/src/api/methods/download/Download.ts

```diff
@@ -238,6 +249,16 @@ export class Download {
                 await asyncPool(maxConcurrentRequests, memberList, createDownloadPromise);
             }
 
+            // Handle failed downloads if no errors were thrown yet
+            if (downloadErrors.length > 0) {
+                throw new ImperativeError({
+                    msg: "Failed to download the following members: \n" + failedMembers.join("\n") + "\n\n" +
```

Comment:
Should the message "Failed to download the following members:" be defined as a constant in `api/constants/ZosFiles.messages.ts`? Just a question, it doesn't need to be

Reply:
We can do that. Would make it easier to keep the test updated if we decide to change it.